### PR TITLE
Add Warm Welcome

### PR DESCRIPTION
**Pull request by kkatzen**
#350
I think we need another pass at the copy in context (for instance, "select location to complete" doesn't make sense in context)

Uses a cookie to only show on first visit but you can override by query param: 
?onboard=true or ?onboard=false

### CODE REVIEW

**Comment by kkatzen** [comment]
aaron points out that we need to check cookie default expiration

**Comment by efregoso** [comment]
For the record, this looks great on both desktop and mobile.  Thank u Krista!

**Comment by kkatzen** [comment]
Updated with some of Chi's suggestions and also removed cookie use so it's only based on URL param. Will add back cookies later.

**Review by vanshkumar — APPROVED** [review]
Very nice! Let's try to get the copy & design fully finalized this week

File: frontend/src/pages/ExploreData/ExploreDataPage.tsx [hunk 1]
```diff
@@ -43,17 +52,39 @@ function ExploreDataPage() {
...
-  const EXPLORE_DATA_ID = "main";
+  // Set up warm welcome onboarding behaviors
+  // TODO(kristak): Add cookies back
+  // const [cookies, setCookie] = useCookies(["name"]);
+  // let showOnboarding = cookies.skipOnboarding !== "true";
+  let showOnboarding = false;
+  if (params[SHOW_ONBOARDING_PARAM] === "true") {
+    showOnboarding = true;
+  }
+  if (params[SHOW_ONBOARDING_PARAM] === "false") {
```
[the remark this answers is not in the document]
Comment (kkatzen) [reply frontend/src/pages/ExploreData/ExploreDataPage.tsx:68]: When we have cookies on, I want onboarding=false to override cookies

**Pull request merged**